app/controllers/locations_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
 
   # GET /locations/:id
   def show
+    @location = Location.find(params[:id])
```

Comment:
This would be unnecessary given you've already declared a `before_action` for `:show`

Reply:
true